### PR TITLE
flake8

### PR DESCRIPTION
**Pull request by bdwyer2**

### CODE REVIEW

File: core/model.py [hunk 1]
```diff
@@ -140,16 +140,14 @@ def load_embedding_model_help(self, model_path, nrof_preprocess_threads=1):
         # Get output tensor
         embeddings = tf.get_default_graph().get_tensor_by_name("embeddings:0")
 
-        return eval_enqueue_op, embeddings, labels_placeholder, \
-               control_placeholder, phase_train_placeholder, \
-               batch_size_placeholder, label_batch, image_placeholder
+        return eval_enqueue_op, embeddings, labels_placeholder, control_placeholder, phase_train_placeholder,\
+               batch_size_placeholder, label_batch, image_placeholder  # noqa
```
Comment (xuhdev) [inline core/model.py]: The following change should be able to dismiss the noqa
```python
        return (eval_enqueue_op, embeddings, labels_placeholder, control_placeholder, phase_train_placeholder,
                batch_size_placeholder, label_batch, image_placeholder)
```

File: core/model.py [hunk 2]
```diff
@@ -140,16 +140,14 @@ def load_embedding_model_help(self, model_path, nrof_preprocess_threads=1):
...
     def load_embedding_model(self, model_path):
-        self.eval_enqueue_op, self.embeddings, self.labels_placeholder, \
-        self.control_placeholder, self.phase_train_placeholder, \
-        self.batch_size_placeholder, self.label_batch, self.image_placeholder = \
-            self.load_embedding_model_help(model_path,
-                                           nrof_preprocess_threads=DEFAULT_PREPROCESS_THREADS)
+        self.eval_enqueue_op, self.embeddings, self.labels_placeholder, self.control_placeholder,\
+        self.phase_train_placeholder, self.batch_size_placeholder, self.label_batch,\
+        self.image_placeholder = self.load_embedding_model_help(model_path,  # noqa
+                                                                nrof_preprocess_threads=DEFAULT_PREPROCESS_THREADS)
```
Comment (xuhdev) [inline core/model.py]: Similarly we can dismiss noqa via

```python
        (self.eval_enqueue_op, self.embeddings, self.labels_placeholder, self.control_placeholder,
         self.phase_train_placeholder, self.batch_size_placeholder, self.label_batch,
         self.image_placeholder) = self.load_embedding_model_help(model_path,
                                                                  nrof_preprocess_threads=DEFAULT_PREPROCESS_THREADS)
```

File: core/src/align/detect_face.py [hunk 3]
```diff
@@ -148,7 +150,7 @@ def conv(self,
         assert c_i % group == 0
         assert c_o % group == 0
         # Convolution for a given input and kernel
-        convolve = lambda i, k: tf.nn.conv2d(i, k, [1, s_h, s_w, 1], padding=padding)
+        convolve = lambda i, k: tf.nn.conv2d(i, k, [1, s_h, s_w, 1], padding=padding)  # noqa
```
Comment (xuhdev) [inline core/src/align/detect_face.py]:
```suggestion
        def convolve(i, k):
            return tf.nn.conv2d(i, k, [1, s_h, s_w, 1], padding=padding)
```

File: core/src/align/detect_face.py [hunk 4]
```diff
@@ -266,34 +270,37 @@ def setup(self):
...
         onet.load(os.path.join(model_path, 'det3.npy'), sess)
-        
-    pnet_fun = lambda img : sess.run(('pnet/conv4-2/BiasAdd:0', 'pnet/prob1:0'), feed_dict={'pnet/input:0':img})
-    rnet_fun = lambda img : sess.run(('rnet/conv5-2/conv5-2:0', 'rnet/prob1:0'), feed_dict={'rnet/input:0':img})
-    onet_fun = lambda img : sess.run(('onet/conv6-2/conv6-2:0', 'onet/conv6-3/conv6-3:0', 'onet/prob1:0'), feed_dict={'onet/input:0':img})
+
+    pnet_fun = lambda img: sess.run(('pnet/conv4-2/BiasAdd:0', 'pnet/prob1:0'), feed_dict={'pnet/input:0': img})  # noqa
+    rnet_fun = lambda img: sess.run(('rnet/conv5-2/conv5-2:0', 'rnet/prob1:0'), feed_dict={'rnet/input:0': img})  # noqa
+    onet_fun = lambda img: sess.run(('onet/conv6-2/conv6-2:0', 'onet/conv6-3/conv6-3:0', 'onet/prob1:0'),  # noqa
+                                    feed_dict={'onet/input:0': img})
```
Comment (xuhdev) [inline core/src/align/detect_face.py]: These noqas can be dismissed if we change them to actual functions (not lambda)

File: core/src/align/detect_face.py (hunk 3 above)
Comment (feihugis) [reply core/src/align/detect_face.py]: Yeah, Hong's suggestion looks like a good practice.

File: core/src/align/detect_face.py (hunk 4 above)
Comment (feihugis) [reply core/src/align/detect_face.py]: Something like below (the indentation need double-check)
```Python
def pnet_fun(img): 
    sess.run(('pnet/conv4-2/BiasAdd:0', 'pnet/prob1:0'), 
             feed_dict={'pnet/input:0':img})
    
def rnet_fun(img): 
    sess.run(('rnet/conv5-2/conv5-2:0', 'rnet/prob1:0'), 
             feed_dict={'rnet/input:0':img})

def onet_fun(image):
    sess.run(('onet/conv6-2/conv6-2:0', 'onet/conv6-3/conv6-3:0', 'onet/prob1:0'), 
             feed_dict={'onet/input:0':img})


```

**Pull request merged**